controllers/error-handling-controller.js

```diff
@@ -1,3 +1,7 @@
+exports.handle404BadPath = (req, res, next) => {
+  res.status(404).send({ msg: "Path not found" });
+};
+
 exports.handle500Status = (err, req, res, next) => {
```

Comment:
It's good to have a console.log(err) in here so that when an error does come through you can trace it yourself easier.